tests/bugfixes/github/test_issue_283.py

```diff
@@ -16,16 +16,15 @@ def check_no_ASAN_UBSAN_errors(self, i, command, got_stderr, expected_stderr):
         Here we want to also check that the two last lines of got_stderr have the expected_stderr
         """
 
-        system_tests.check_no_ASAN_UBSAN_errors(self, i, command, got_stderr, expected_stderr)
+        check_no_ASAN_UBSAN_errors(self, i, command, got_stderr, expected_stderr)
         self.assertListEqual(expected_stderr.splitlines(), got_stderr.splitlines()[-2:])
 
-    filename = os.path.join("$data_path", "pocIssue283.jpg")
+    filename = p("$data_path/pocIssue283.jpg")
```

Comment:
Please change the name of the function `p` to something more meaningful like: platformPath, uniformPath, convertPath, etc. Anything you come up will be better than simply `p` :wink:

Reply:
I wanted to keep it short and concise to not hamper readability, but I agree, `p` is not the best name. What do you think about simply calling it `path`? That would not be awfully long and would further explain that the line is a path.

Reply:
That also works for me ;). I'll approve once you change it.